server/chat-commands/info.ts

```diff
@@ -541,7 +541,9 @@ export const commands: ChatCommands = {
 				if (format?.onModifySpecies) {
 					pokemon = format.onModifySpecies.call({dex} as Battle, pokemon) || pokemon;
 				}
-				let displayedTier = pokemon.tier;
+				let displayedTier = room?.dataCommandTierDisplay === 'tiers' ? pokemon.tier :
+					room.dataCommandTierDisplay === 'doubles tiers' ? pokemon.doublesTier :
```

Comment:
Is it possible for `room` to be `null`? If so, `room.dataCommandTierDisplay` is wrong, if not, `room?.dataCommandTierDisplay` is wrong.

Reply:
Looks like yes.

```suggestion
				let displayedTier = room?.dataCommandTierDisplay === 'tiers' ? pokemon.tier :
					room?.dataCommandTierDisplay === 'doubles tiers' ? pokemon.doublesTier :
```